.github/workflows/edit.yml

```diff
@@ -0,0 +1,16 @@
+name: edit
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
```

Comment:
Action docs suggest also including "edited" in this list. Any reason to exclude it?